packages/server/src/sdk/app/views/index.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  DocumentType,
+  SEPARATOR,
+  UNICODE_MAX,
+  context,
+} from "@budibase/backend-core"
+import { ViewV2 } from "@budibase/types"
+import * as utils from "../../../db/utils"
+
+export async function fetch() {
+  const db = context.getAppDB()
+
+  const startKey = `${DocumentType.VIEW}${SEPARATOR}`
+  const response = await db.allDocs({
+    startkey: startKey,
+    endkey: `${startKey}${UNICODE_MAX}`,
+    include_docs: true,
+  })
+
+  return response.rows.map(r => r.doc)
+}
+
+export async function findByTable(tableId: string) {
+  const db = context.getAppDB()
+
+  const startKey = utils.viewIDPrefix(tableId)
+  const response = await db.allDocs({
+    startkey: startKey,
+    endkey: `${startKey}${UNICODE_MAX}`,
+    include_docs: true,
+  })
+
+  return response.rows.map(r => r.doc)
+}
+
+export async function get(viewId: string) {
+  const db = context.getAppDB()
+  const result = await db.get(viewId)
+  return result
```

Comment:
I believe this will return `any`. Could we type it?

Reply:
Typed! I just realised our types are returning `get<T>(id?: string): Promise<T | any>`. This `any` will always ignore the T, so we should probably fix this. I will do it in another branch, as it might have plenty of changes

Reply:
https://github.com/Budibase/budibase/pull/11259

Reply:
Nice one @adrinr. Initially I had meant to add typing to the functions inside this module only basically casting the `any`, but your other PR is a better way of handling this across the board 👍